scripts/globals/items/bottle_of_hysteroanima.lua

```diff
@@ -0,0 +1,29 @@
+-----------------------------------------
+-- ID: 5262
+-- Item: Bottle Of Hysteroanima
+-- Item Effect: Amnesia
+-----------------------------------------
+require("scripts/globals/msg")
+require("scripts/globals/status")
+require("scripts/globals/player")
+-----------------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target, player)
+    local result = 0
+    if target:getSystem() ~= xi.ecosystem.EMPTY then -- Empty
+        result = xi.msg.basic.ITEM_UNABLE_TO_USE
+    elseif target:checkDistance(player) > 10 then
+        result = xi.msg.basic.TOO_FAR_AWAY
+    end
+
+    return result
+end
+
+item_object.onItemUse = function(target, player)
+    target:delStatusEffectSilent(xi.effect.AMNESIA)
+    target:addStatusEffect(xi.effect.AMNESIA, 1, 0, math.random(25,32))
+    player:messageBasic(xi.msg.basic.GAINS_EFFECT_OF_STATUS, xi.effect.AMNESIA, 0, target)
```

Comment:
I don't think SE does this using an actual amnesia effect. 

> Using a Hysteroanima on an Empty foe in Promyvion will prevent it from using any TP abilities for 30 seconds, similar to an Amnesia effect. (The abilities will appear to be readied, but will not execute.) Regular attacks may still be used.

I think if it were actual amnesia, they'd have said so there. The fact they still ready also indicates it's not amnesia.

Likewise I think the other 2 anima's are different but I'd need to check.

Reply:
I did not catch that. That is beyond my coding knowledge to do something like that. I can just remove all 3 bottles and make it a TODO. lmk

Reply:
I am hoping someone else chimes in with a suggestion/offer to assist before we give up on it entirely, as these 3 items have lain unusable for many years now.

Reply:
If we make a new modid for terroanima we can check it in core and stop WS after ready. likely it uses the same underlying mechanism as "too far away" just without the message. It'd need a packet cap to see what exactly it does for the action packet, and that probably isn't all that hard to do.

It is notable that real "amnesia" is actually worse here, because readying TP moves wastes time that could be used meleeing, so you do want this feature.

We just need to add an invisible/no name status to the mob that applies the new mod

Reply:
I would say let it through as a big todo, as frank said this is out of the scope of his comfort due to ability. He can obviously remove them for now if you arn't happy. But to be honest having the items do something in the meantime thats close as a TODO is better than nothing isn't it?

Reply:
@The-Aerec while you're entitled to you're opinion I was trying to get him an assist -not make him code outside his comfort zone- before leaving the items out. We will not ever absorb something we know to be incorrect. Is effectively been a todo for like 6 years if not longer. So pardon me if the PR waits a little while longer. Its not about me being happy.

Reply:
> @The-Aerec while you're entitled to you're opinion I was trying to get him an assist -not make him code outside his comfort zone- before leaving the items out. We will not ever absorb something we know to be incorrect. Is effectively been a todo for like 6 years if not longer. So pardon me if the PR waits a little while longer. Its not about me being happy.

@TeoTwawki - I deleted the comment because I re-read it and it came out sounding wrong. I understand.

Reply:
I am interested in implementing the Hysteroanima/Psychoanima, mostly because I know how to do that. Psychoanima i think is beyond my knowledge of the current codebase, maybe in some time but not now.

I need to know some things about each to make the backend work for these items:

- What is the intimidation rate of Psychoanima on someone who isn't wearing intimidation vs Empty gear (wiki claims it's 100%.... but I would think it may not actually be ALL attacks, just enough to eyeball it as "all")
- Can the *anima items "debuff" be removed by Winds of Promyvion from another empty? if so, what is the action packet for that?

We can make Hysteroanima work in the backend pretty easy, but I wish to understand if the *anima items are actually a "debuff" on the mob or not

Reply:
After further review as seen in: https://www.youtube.com/watch?v=M5roBxP-YTE&ab_channel=GrapplingWithGamesGWG

Hysteroanima actually uses the full ability but returns 0 and a msg.basic.NONE

Reply:
Yup. Looks right to me, I just reversed "too far away" interrupts and that is _not_ an interrupt. Just an ability that does nothing but animate.

Reply:
@TeoTwawki we have moved the Hysteroanima to use the newly created HYSTERIA effect. This will still allow for the core function of applying a random debuff timer while cleaning up the messages and abilities used during it. When HYSTERIA is applied and in effect, the mob will show that it does the mobkill, do the animation for the mobskill, and then return nothing as a message and does not put in any damage. Please see the video that Frank linked above. 

@WinterSolstice8 based on what I've seen in captures from the fights, the intimidate is 100% proc thus empty gear calc are not needed to determine the effectiveness of the Psychoanima. 

Beyond this, due to the limited testing capabilities as of now and the limited capture packets from zones with multiple empty mobs having these items used one them, we cannot say for certain. This is primarily due to how difficult these items are to obtain so players naturally avoid using them on mobs in groups when going through promies. Thus, in a standard gameplay loop this occurrence would more than likely never happen.

Reply:
So I went and checked myself just so I could see the packet:
```
[09:23:05] [Redacted] bottle of hysteroanima → Redacted
[09:23:22] [Apex・Woeful・Lamenter] Auroral Drape → Redacted
[09:23:22] {
[09:23:22]   actor_id = 16867673,
[09:23:22]   category = 7,
[09:23:22]   param = 24931,
[09:23:22]   recast = 0,
[09:23:22]   target_count = 1,
[09:23:22]   targets = { {
[09:23:22]       action_count = 1,
[09:23:22]       actions = { {
[09:23:22]           add_effect_animation = 0,
[09:23:22]           add_effect_effect = 0,
[09:23:22]           add_effect_message = 0,
[09:23:22]           add_effect_param = 0,
[09:23:22]           animation = 0,
[09:23:22]           effect = 0,
[09:23:22]           has_add_effect = false,
[09:23:22]           has_spike_effect = false,
[09:23:22]           knockback = 0,
[09:23:22]           message = 43,
[09:23:22]           param = 1220,
[09:23:22]           reaction = 8,
[09:23:22]           spike_effect_animation = 0,
[09:23:22]           spike_effect_effect = 0,
[09:23:22]           spike_effect_message = 0,
[09:23:22]           spike_effect_param = 0,
[09:23:22]           stagger = 2,
[09:23:22]           unknown = 0
[09:23:22]         } },
[09:23:22]       id = 43455
[09:23:22]     } },
[09:23:22]   unknown = 25972
[09:23:22] }
[09:23:23] {
[09:23:23]   actor_id = 16867673,
[09:23:23]   category = 7,
[09:23:23]   param = 28787,
[09:23:23]   recast = 0,
[09:23:23]   target_count = 1,
[09:23:23]   targets = { {
[09:23:23]       action_count = 1,
[09:23:23]       actions = { {
[09:23:23]           add_effect_animation = 0,
[09:23:23]           add_effect_effect = 0,
[09:23:23]           add_effect_message = 0,
[09:23:23]           add_effect_param = 0,
[09:23:23]           animation = 0,
[09:23:23]           effect = 0,
[09:23:23]           has_add_effect = false,
[09:23:23]           has_spike_effect = false,
[09:23:23]           knockback = 0,
[09:23:23]           message = 0,
[09:23:23]           param = 0,
[09:23:23]           reaction = 8,
[09:23:23]           spike_effect_animation = 0,
[09:23:23]           spike_effect_effect = 0,
[09:23:23]           spike_effect_message = 0,
[09:23:23]           spike_effect_param = 0,
[09:23:23]           stagger = 2,
[09:23:23]           unknown = 0
[09:23:23]         } },
[09:23:23]       id = 16867673
[09:23:23]     } },
[09:23:23]   unknown = 25972
[09:23:23] }
```

It "finishes" with a cat 7 and no animation. I don't think it's possible for lua to modify the category right now with no binding, plus this would incur significant jank that needs a closer look. leave a todo about changing to category 7 and no animation in the hysteroanima file

Reply:
after the todo is added I'll slap approve button